tooling/create-tauri-app/src/helpers/update-tauri-conf.ts

```diff
@@ -2,15 +2,18 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-License-Identifier: MIT
 
-import { readFileSync, writeFileSync } from 'fs'
+import { readFileSync, writeFileSync, mkdirSync } from 'fs'
 import { join } from 'path'
 import { TauriBuildConfig } from '../types/config'
 
 export function updateTauriConf(
   appDirectory: string,
   cfg: TauriBuildConfig
 ): void {
-  const tauriConfPath = join(appDirectory, 'src-tauri', 'tauri.conf.json')
+  const tauriConfDir = join(appDirectory, 'src-tauri')
+  mkdirSync(tauriConfDir, { recursive: true });
```

Comment:
I think this is not needed considering this function should only be called after `tauri init` execution finishes. Am I wrong @jbolda ?

Reply:
That is my understanding. `tauri init` should be creating the folder and the `tauri.conf.json` file. @ikey4u did you run into an error that prompted this PR?